File: read_files.py
files = [
    r'C:\Users\khush\75her\frontend\app\(dashboard)\layout.tsx',
    r'C:\Users\khush\75her\frontend\lib\sidebar-context.tsx',
    r'C:\Users\khush\75her\frontend\components\patient-portal\vitals-overview.tsx',
    r'C:\Users\khush\75her\frontend\components\doc-monitor\alert-panel.tsx',
    r'C:\Users\khush\75her\frontend\components\doc-monitor\approval-queue.tsx',
]
for f in files:
    print('===FILE:', f, '===')
    try:
        with open(f, 'r', encoding='utf-8') as fh:
            print(fh.read())
    except Exception as e:
        print('ERROR:', e)
    print('===END===')

File: read_files.js
const fs = require('fs');
const base = 'C:/Users/khush/75her/frontend/';
const files = [
  'app/(dashboard)/layout.tsx',
  'components/layout/sidebar.tsx',
  'components/layout/topbar.tsx',
  'components/patient-portal/vitals-overview.tsx',
  'components/doc-monitor/alert-panel.tsx',
  'components/doc-monitor/approval-queue.tsx',
  'components/common/blockchain-badge.tsx',
  'lib/sidebar-context.tsx',
];
for (const f of files) {
  console.log('\n=== FILE: ' + f + ' ===');
  try {
    console.log(fs.readFileSync(base + f, 'utf8'));
  } catch(e) {
    console.log('ERROR: ' + e.message);
  }
}
